chance/chance.d.ts

```diff
@@ -115,7 +116,8 @@ declare namespace Chance {
         mixin(desc: MixinDescriptor): any;
         pad(num: number, width: number, padChar?: string): string;
         pick<T>(arr: T[]): T;
-        pick<T>(arr: T[], count: number): T[];
+        pick<T>(arr: T[], count: number): T | T[];
```

Comment:
from the documentation the return value seems to be always an array. why is it not an array?